bibtex/bib/quantum.bib

```diff
@@ -90,6 +101,23 @@ @Article{BHK05
   Url                      = {http://dx.doi.org/10.1103/PhysRevLett.95.010503}
 }
 
+@Article{BBB+92,
+  Title                    = {Experimental quantum cryptography},
+  Author                   = {Bennett, CharlesH. and Bessette, Franï¿½ois and Brassard, Gilles and Salvail, Louis and Smolin, John},
```

Comment:
there's something wrong with the special character here, those should be written in {curly brackets}, like
{\'e}